.ci/pipelines/release_zeebe_java11.groovy

```diff
@@ -41,7 +41,7 @@ spec:
           cpu: 8
           memory: 32Gi
     - name: golang
-      image: golang:1.15.15
+      image: golang:1.17.11
```

Comment:
:x: Please leave to 1.15.15 here, since the Java 11 release pipeline is only for versions < 1.3 (and these will not be updated to run on 1.17 AFAIK).